core/modules/filters.js

```diff
@@ -30,7 +30,8 @@ function parseFilterOperation(operators,filterString,p) {
 		operator = {};
 		// Check for an operator prefix
 		if(filterString.charAt(p) === "!") {
-			operator.prefix = filterString.charAt(p++);
+			operator.prefix = filterString.charAt(p++); // remove?
```

Comment:
But there's lots of core code using operator.prefix instead, isn't there? I agree that operator.negate is better but we should apply it consistently if we're going to introduce it.